### PR TITLE
[MM-33377] Move AWS invoke credentials to env variables

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
In production, these credentials will be provided by the cloud team and configured via env variables.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-33377

### CODE REVIEW

File: server/plugin.go [hunk 1]
```diff
@@ -64,7 +65,16 @@ func (p *Plugin) OnActivate() error {
...
+	accessKey := os.Getenv("APPS_INVOKE_AWS_ACCESS_KEY")
+	if accessKey == "" {
+		mm.Log.Warn("APPS_INVOKE_AWS_ACCESS_KEY is not set. AWS apps won't work.")
+	}
+	secretKey := os.Getenv("APPS_INVOKE_AWS_SECRET_KEY")
+	if secretKey == "" {
+		mm.Log.Warn("APPS_INVOKE_AWS_SECRET_KEY is not set. AWS apps won't work.")
+	}
+
+	awsClient := aws.NewAWSClient(accessKey, secretKey, &mm.Log)
```
Comment (iomodo) [inline server/plugin.go:77]: if `APPS_INVOKE_AWS_ACCESS_KEY` and `APPS_INVOKE_AWS_SECRET_KEY` are empty `aws.NewAWSClient` may still create a credentials. It is reading from the "default env vars" `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. So either we get rid of the previous behaviour or correct the messages or get rid of this new env vars.

**Review by iomodo — CHANGES_REQUESTED** [review]

File: server/plugin.go (hunk 1 above)
Comment (hanzei) [reply server/plugin.go:77]: I'm inclined to get rid of get rid of the previous behavior and don't use `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. Given that we are using two sets of credentials, I think it's better to be explicit. @iomodo What do you think?

Comment (iomodo) [reply server/plugin.go:77]: Agree

Comment (hanzei) [reply server/plugin.go:77]: Changed via c37e9d8

**Review by iomodo — APPROVED** [review]

**Review by levb — APPROVED** [review]

**Pull request merged**